modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini

```diff
@@ -276,6 +276,8 @@ GBP-SONIA-OIS-COMPOUND = GBP-SONIA
 GBP-SONIA-OIS Compound = GBP-SONIA
 GBP-SONIA-WMBA-COMPOUND = GBP-SONIA
 
+GBP-TERMSONIA = GBP-TERMSONIA
```

Comment:
ISDA name seems to be "GBP-SONIA ICE Term" or "GBP-SONIA Refinitiv Term"

Reply:
There is indeed the possibility of several term rates published by different providers. Should we include the provider in the name? We have not done it in the past (there was only one LIBOR, one EURIBOR, etc.).
A name like "GBP-SONIAICETERM-1M" or "GBP-SONIAREFINITIVTERM-1M"would be more explicit.